rest-service/manager_rest/cluster_status_manager.py

```diff
@@ -455,12 +540,24 @@ def _save_report(report_path, report_dict):
     with open(report_path, 'w') as report_file:
         json.dump(report_dict, report_file)
 
+
+def _verify_syncthing_status():
+    if not (ha_utils and ha_utils.is_clustered()):
+        return
+    syncthing_status, _ = get_syncthing_status()
+    if syncthing_status == NodeServiceStatus.INACTIVE:
+        raise manager_exceptions.FileSyncServiceError(
+            "Can't update the status report when Syncthing is not healthy"
```

Comment:
Well syncthing is the way we create the communication line between the managers, but the issue is that this manager is not healthy member of the cluster

Reply:
So not sure about the name

Reply:
You are not sure about what name? The exception name or the message?

Reply:
FileSyncServiceError name